app/views/staff/organisations/show.html.haml

```diff
@@ -15,9 +15,8 @@
           = t("create_programme", scope: "page_content.organisation")
 
         - @funds.each do |fund|
-          = link_to t("form.button.activity.new_child", name: fund.title),
-            organisation_activity_children_path(@organisation_presenter, fund.id),
-            method: :post, class: "govuk-button govuk-!-margin-right-4"
+          = form_tag organisation_activity_children_path(@organisation_presenter, fund), method: :post do
+            = submit_tag t("form.button.activity.new_child", name: fund.title), class: "govuk-button govuk-!-margin-right-4"
```

Comment:
You could use [`button_to`](https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to) here

Reply:
I always forget about `button to` ! 🤦

Reply:
Was this meant as an agreement to use `button_to` now, or in the future (but not changing the code now)?

Reply:
@CristinaRO I'm going to change to using `button_to` now.